collector/rapl_linux.go

```diff
@@ -26,17 +26,24 @@ import (
 	"github.com/go-kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/procfs/sysfs"
+	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 type raplCollector struct {
 	fs     sysfs.FS
 	logger log.Logger
+
+	joulesMetricDesc *prometheus.Desc
 }
 
 func init() {
 	registerCollector("rapl", defaultEnabled, NewRaplCollector)
```

Comment:
`rapl` could be a const since it is used multiple times in this code.

Reply:
Fixed.